src/balance.js

```diff
@@ -1,17 +1,32 @@
 const { BN } = require('./setup');
 
-async function balanceCurrent (account) {
-  return new BN(await web3.eth.getBalance(account));
+class Tracker {
+  constructor (acc) {
+    this.account = acc;
+    this.balances = [];
+  }
+  async delta () {
+    const current = await this.get();
+    return current.sub(this.balances[this.balances.length - 2]);
+  }
+  async get () {
+    const bal = new BN(await web3.eth.getBalance(this.account));
+    this.balances.push(bal);
```

Comment:
Do we need to keep all past balances? It feels like keeping only the last one would be enough.